projects/1.21/assets/flapjacks-economy/jackseconomy/lang/zh_cn.json

```
@@ -0,0 +1,150 @@
+{
+  "block.jackseconomy.exporter": "出口者",
+  "block.jackseconomy.importer": "进口者",
+  "item.jackseconomy.penny": "1分硬币",
+  "item.jackseconomy.nickel": "5分硬币",
+  "item.jackseconomy.dime": "10分硬币",
+  "item.jackseconomy.quarter": "25分硬币",
+  "item.jackseconomy.dollar_bill": "1元纸币",
+  "item.jackseconomy.five_dollar_bill": "5元纸币",
+  "item.jackseconomy.ten_dollar_bill": "10元纸币",
+  "item.jackseconomy.twenty_dollar_bill": "20元纸币",
+  "item.jackseconomy.fifty_dollar_bill": "50元纸币",
+  "item.jackseconomy.hundred_dollar_bill": "100元纸币",
+  "item.jackseconomy.thousand_dollar_bill": "1000元纸币",
+  "item.jackseconomy.wallet": "钱包",
+  "item.jackseconomy.basic_wallet": "基础钱包",
+  "item.jackseconomy.intermediate_wallet": "中级钱包",
+  "item.jackseconomy.advanced_wallet": "高级钱包",
+  "item.jackseconomy.the_phat_wallet": "非常好的钱包",
+  "item.jackseconomy.check": "支票",
+  "item.jackseconomy.exporter_manifest": "出口清单",
+  "item.jackseconomy.importer_manifest": "进口清单",
+  "item.jackseconomy.golden_exporter_manifest": "金出口清单",
+  "item.jackseconomy.empty_exporter_manifest": "空出口清单",
+  "item.jackseconomy.empty_importer_manifest": "空进口清单",
+  "block.jackseconomy.penny_stack": "1分硬币堆",
+  "block.jackseconomy.nickel_stack": "5分硬币堆",
+  "block.jackseconomy.dime_stack": "10分硬币堆",
+  "block.jackseconomy.quarter_stack": "25分硬币堆",
+  "block.jackseconomy.dollar_bill_stack": "1元纸币堆",
+  "block.jackseconomy.five_dollar_bill_stack": "5元纸币堆",
+  "block.jackseconomy.ten_dollar_bill_stack": "10元纸币堆",
+  "block.jackseconomy.twenty_dollar_bill_stack": "20元纸币堆",
+  "block.jackseconomy.fifty_dollar_bill_stack": "50元纸币堆",
+  "block.jackseconomy.hundred_dollar_bill_stack": "100元纸币堆",
+  "block.jackseconomy.thousand_dollar_bill_stack": "1000元纸币堆",
+  "itemGroup.jackseconomy": "Jack的经济",
+  "jackseconomy.click_to_select": "点击选择！",
+  "jackseconomy.selected": "已选中",
+  "jackseconomy.search": "搜索…",
+  "jackseconomy.sell_price": "售价(%s): %s",
```

```suggestion
  "jackseconomy.sell_price": "售价（%s）： %s",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

除非ui实在太挤，否则还是建议使用全角标点